Profile/Profile/Data/ProfileRepository.swift

```diff
@@ -71,8 +71,8 @@ public class ProfileRepository: ProfileRepositoryProtocol {
             ProfileEndpoint.logOut(refreshToken: refreshToken, clientID: config.oAuthClientId)
         )
         storage.clear()
-        await downloadManager.deleteAllFiles()
-        coreDataHandler.clear()
+        //await downloadManager.deleteAllFiles()
+        //coreDataHandler.clear()
```

Comment:
@eyatsenkoperpetio Do we need to keep this as well if we want to keep downloaded files?